web/modules/custom/farm_nfa/farm_nfa.install

```diff
@@ -273,3 +274,69 @@ function farm_nfa_update_9019(&$sandbox) {
   $process = Drush::drush(Drush::aliasManager()->getSelf(), 'config-import', ['--partial'], ['source' => $config_path]);
   $process->run();
 }
+
+/**
+ * save the taxonomy terms for forest_vegetation and forest_type taxonomy types.
+ */
+function farm_nfa_update_9020(&$sandbox) {
+  $vocab_terms = [
+    'forest_vegetation' => [
+      'Tropical High Forest',
+      'Woodland',
+      'Bush',
+    ],
+    'forest_type' => [
```

Comment:
Do we need forest_type in this hook?

Reply:
yea @peacog correct, i forgot we changed our approach and started setting the forest_type from the farm_nfa_forest.module manually . have updated this accordingly, thanks !